content/usage/_index.md

```diff
@@ -39,27 +39,34 @@ This command will open configmap specs in vim editor. Do the required changes, s
 
 
 <h3 class="section-head" id="h-check-health"><a href="#h-check-health">Check BotKube health</a></h3>
-Run **@BotKube ping** to the Slack channel where BotKube is added. The BotKube will respond you with message **PONG** if it is running. Else check the deployment in Kubernetes cluster in the **botkube** namespace.
+Run **@BotKube ping** to the Slack channel where BotKube is added. The BotKube will respond you with the **PONG** message from all the configured clusters. Use `--cluster-name` flag to get response from the cluster mentioned in the flag. Else check the deployment in Kubernetes cluster in the **botkube** namespace.
 
 ![ping](/images/ping.png)
 
 
 <h3 class="section-head" id="h-notifer-stop"><a href="#h-notifier-stop">Disable Slack notifications</a></h3>
 If you want to stop receiving Slack notifications from BotKube, run
 **@BotKube notifier stop**
-to the Slack channel where BotKube is added. You will no longer receive notifications from the BotKube
+in the configured Slack channel where BotKube is added. You will no longer receive notifications from the BotKube

```

Comment:
from

Reply:
Made requested changes. Updated in the next commit.